src/components/Book.jsx

```diff
@@ -4,37 +4,63 @@ import PropTypes from 'prop-types';
 import { deleteBook, fetchBooks } from '../redux/books/booksSlice';
 
 const Book = (props) => {
-  // eslint-disable-next-line camelcase
-  const { item_id, title, author } = props;
+  const {
+    itemId, title, author, category,
+  } = props;
   const dispatch = useDispatch();
 
   const handleClick = async () => {
-    await dispatch(deleteBook(item_id));
+    await dispatch(deleteBook(itemId));
     await dispatch(fetchBooks());
   };
   return (
-    <li className="book">
-      <p className="book-title">{title}</p>
-      <p className="author-name">
-        {' '}
-        by
-        {author}
-      </p>
-      <div>
-        <button type="button">Comments</button>
-        <span>|</span>
-        <button type="button" onClick={handleClick}>Remove</button>
-        <span>|</span>
-        <button type="button">Edit</button>
+    <li>
+      <div className="book">
+        <div className="book-content">
+          <div className="book-details">
+            <p className="book-category">{category}</p>
+            <p className="book-title">{title}</p>
+            <p className="author-name">{author}</p>
+            <div className="action-buttons">
+              <button className="btn-outline" type="button">Comments</button>
+              <span>|</span>
+              <button className="btn-outline" type="button" onClick={handleClick}>Remove</button>
+              <span>|</span>
+              <button className="btn-outline" type="button">Edit</button>
```

Comment:
- [ ] As I mentioned in my previous comment, you should follow the design properly, because as I can see you are not using the Correct `font-family`, `font-colour` etc. which makes your design `inconsistent` and looks unprofessional. Please have a look at the attached image and table and don't forget to align the items correctly.

### Requirement
![Screenshot from 2023-09-09 21-59-56](https://github.com/mulugithub/bookstore/assets/66476329/dbfd71d2-7a0d-47df-911c-07801c432843)

### Design Comparison
<table>
<tr>
<th>Zeplin's Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-09-09 22-02-19](https://github.com/mulugithub/bookstore/assets/66476329/3aaf1a69-224b-4a24-b6b4-c435c97f6ee6)

</td>
<td>

![Screenshot from 2023-09-09 23-44-36](https://github.com/yusufyusie/Bookstore-CMS/assets/66476329/737095d0-db9c-42dc-b830-4b6cf651cec9)

</td>
</tr>
</table>